### ISSUE TITLE
Tags are not accessible with keyboard (WCAG)

### ISSUE THREAD

**Issue by shed25** [vueform/multiselect#422]
### Version

* Vue version: 3.3.9

### Description

In mode tags user cannot navigate through the tags using tab or shift+tab

### Demo

https://jsfiddle.net/brmwp0zv/2/
